packages/esm-patient-conditions-app/src/conditions/conditions-form.workspace.tsx

```diff
@@ -102,6 +86,7 @@ const ConditionsForm: React.FC<ConditionFormProps> = ({
     <FormProvider {...methods}>
       <Form className={styles.form} onSubmit={methods.handleSubmit(onSubmit, onError)}>
         <ConditionsWidget
+          schema={schema}
```

Comment:
I'm now passing down this type annotation as props. It doesn't feel like the best approach.